cmd/secrets-webhook/Makefile

```diff
@@ -0,0 +1,18 @@
+help: ## Display this help.
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+.PHONY: test
+test: ## run unit tests
+	@echo "\n🛠️  Running unit tests..."
+	go test ./...
+
+.PHONY: build
+build: ## build binaries
```

Comment:
Can I suggest running steps including `go fmt`, `test` and `go vet` as separate make targets but as dependancies to this target.